docker/config/crontabber.env

```diff
@@ -0,0 +1,10 @@
+# Crontabber configuration file covering behavior configuration for
+# crontabber
+
+# NOTE: This file shouldn't contain any environment-specific or secret
+# configuration.
+
+# ----------------------------------------------------------------------------------
+# from socorro/crontabber
+# ----------------------------------------------------------------------------------
+crontabber.class-BugzillaCronApp.days_into_past=0
```

Comment:
I literally pulled all the configuration from -prod and broke it up into behavioral configuration files. I didn't then go through all the keys and see which ones we could remove.

Sounds like a good pass for someone to do.